packages/mds-types/trip.ts

```diff
@@ -25,13 +25,13 @@ export interface TripsStats {
   mystery_examples: { [key: string]: UUID[] }
 }
 
-export interface TripMetadata {
+export type TripMetadata<T extends any = {}> = {
```

Comment:
I think this could simply be `<T = {}>`

Reply:
Fixed in 1356ea2456b9643807aecabf8316fd49edcbd995